Roslyn.sln

```diff
@@ -452,689 +454,1115 @@ Global
 	EndGlobalSection
 	GlobalSection(SolutionConfigurationPlatforms) = preSolution
 		Debug|Any CPU = Debug|Any CPU
+		Debug|x64 = Debug|x64
```

Comment:
I suspect you didn't intend to have this happen (and it'll probably cause funky things too.)